flow/designs/asap7/mock-array/fastroute.tcl

```diff
@@ -0,0 +1,2 @@
+set_global_routing_layer_adjustment $env(MIN_ROUTING_LAYER)-$env(MAX_ROUTING_LAYER) 0.45
```

Comment:
what is this magic number 0.45

Reply:
It was the number that made it pass on CI. We can investigate this further, but I would like to merge the PR as the update contains metric updates for secure CI.

Reply:
ok. add the comments to explain to a commit in this PR and the merging is fine w me.